Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1

```diff
@@ -9,6 +9,7 @@ $ErrorActionPreference = 'Stop'
 
 $script:deploymentSettingsFiles = @()
 $script:deploymentSettingsOverrides = @{ }
+Import-Module -Force -Scope Global "$PSScriptRoot/Ed-Fi-ODS-Implementation/logistics/scripts/modules/path-resolver.psm1"
```

Comment:
wouldn't importing `path-resolver` here prevent you from setting `$env:toolsPath` in your code below?

Reply:
OK fixed it . Thanks